session.go

```diff
@@ -94,8 +106,25 @@ func (s *Session) startGC() {
 	}()
 }
 
+// Check if the remote IP is in a trusted network.
+func (s *Session) isRequestFromTrustedNetwork(ctx *fasthttp.RequestCtx) bool {
+	for _, trustedNet := range s.trustedNets {
+		if trustedNet.Contains(ctx.RemoteIP()) {
+			return true
+		}
+	}
+	return false
+}
+
+// Return true if X-Forwarded-Proto is provided and set to https.
+func isForwardedProtoHTTPS(ctx *fasthttp.RequestCtx) bool {
+	return bytes.Equal(ctx.Request.Header.Peek("X-Forwarded-Proto"), []byte("https"))
```

Comment:
This line make an extra allocation, define `[]byte("https")` outsite the function, please

Reply:
If you don't mind, move this function to final of file

Reply:
ack. Will do.